doc/examples/bingham_fit_fodf.py

```diff
@@ -0,0 +1,126 @@
+"""
+=======================================
+Fitting Bingham distributions to an ODF
+=======================================
+
+This example demonstrates how to decompose an ODF into a combination
+of Bingham distributions [Riffert2014]_.
+
+First, load useful imports.
+"""
+
+import numpy as np
+from dipy.direction.bingham import (bingham_fit_odf, bingham_odf,
+                                    bingham_fiber_density,
+                                    bingham_fiber_spread,
+                                    bingham_orientation_dispersion
+                                    )
+
+from dipy.sims.voxel import multi_tensor_odf
+from dipy.data import get_sphere
+from dipy.viz import window, actor
+
+
+"""
```

Comment:
old formatting style, Can you update this tutorial by following the new formatting style. Check your other example